Views/Shared/Scripts/_UtcToLocal.cshtml

```diff
@@ -0,0 +1,12 @@
+﻿<script src="~/lib/moment.js/moment.min.js"></script>
```

Comment:
Moment is really big library (and is no longer updated), you might want to swap to something else like https://github.com/iamkun/dayjs.

Reply:
Or even better take just one function you need from moment/day and use it.